pkg/test/framework/components/echo/kube/templates/deployment.yaml

```diff
@@ -76,6 +76,7 @@ spec:
   (ne (index $subset.Annotations "sidecar.istio.io/inject") "false")
   (ne (index $subset.Annotations "inject.istio.io/templates") "grpc")
   ($.OverlayIstioProxy)
+  (not (and $.OpenShift (eq $.Service "tproxy")))
```

Comment:
I also found a bug in merging customized istio-proxy's security context with the injection template, but I will address this problem separately.